src/focus-trap-react.js

```diff
@@ -87,9 +87,10 @@ class FocusTrap extends React.Component {
 
   /** Update the previously focused element with the currently focused element. */
   updatePreviousElement() {
-    if (typeof document !== 'undefined') {
-      this.previouslyFocusedElement = document.activeElement;
+    if (typeof this.props.focusTrapOptions.document !== 'undefined') {
+      this.previouslyFocusedElement = this.props.focusTrapOptions.document.activeElement;
     }
```

Comment:
Maybe this, to cover the case where the option isn't specified?
```suggestion
    const doc = this.props.focusTrapOptions.document || document;
    if (doc) {
      this.previouslyFocusedElement = doc.activeElement;
    }
```